### PR TITLE
Feature/password expiration

### PR DESCRIPTION
**Pull request by Nainterceptor**
Checklist

- [x] I've added a mention in CHANGELOG.md

### CODE REVIEW

File: components/form/password.vue [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    <input
+      v-validate="'required'"
+      v-bind="$attrs"
+      type="password"
+      class="input"
+      :class="{ 'is-danger': !isValid || isInvalid }"
+      :value="val || value"
+      name="password"
+      data-vv-as="Mot de passe"
+      :placeholder="'Tapez votre mot de passe'"
```
Comment (Scttpr) [inline components/form/password.vue]: Directive not required here ?

File: components/form/password.vue [hunk 2]
```diff
@@ -0,0 +1,63 @@
+<template>
+  <div>
+    <input
+      v-validate="'required'"
+      v-bind="$attrs"
+      type="password"
+      class="input"
```
Comment (Scttpr) [inline components/form/password.vue]: Is it needed ? If it's for CSS, I would take benefits from scoped attributes and use basic selector

File: components/form/password.vue [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    minLength: {
+      type: Number,
+      default: 8,
+    },
+    isInvalid: {
+      type: Boolean,
+      default: false,
+    },
+  },
+  data: () => ({
```
Comment (Scttpr) [inline components/form/password.vue]: I think it's weird not to give full ownership to the component to manage is local data and state. I would let value management outside of parent/child relationship.

File: pages/login.vue [hunk 4]
```diff
@@ -97,15 +97,21 @@ export default {
         switch (status) {
           case 404:
             this.$toast.error('Impossible de se connecter, l\'utilisateur n\'existe pas. '
-              + 'Merci de vérifier votre identifiant.\n'
-              + 'Si le problème persiste, contactez nous pour confirmer votre identifiant.');
+              + 'Merci de vérifier votre identifiant.');
```
Comment (Scttpr) [inline pages/login.vue]: I would empty input fields here, at least email and password ?

File: pages/rating.vue [hunk 5]
```diff
@@ -72,11 +72,11 @@
             label="Avez-vous des suggestions pour améliorer cette offre de mobilité ?"
           >
             <textarea
+              v-model="fields.message"
```
Comment (Scttpr) [inline pages/rating.vue]: Guess it's eslint work here :p

File: components/form/password.vue (hunk 1 above)
Comment (Nainterceptor) [reply components/form/password.vue]: You're right, changed in password component + legacies of edit-account file

File: components/form/password.vue (hunk 2 above)
Comment (Nainterceptor) [reply components/form/password.vue]: It's bulma class :)

File: pages/login.vue (hunk 4 above)
Comment (Nainterceptor) [reply pages/login.vue]: Not needed, we want to give opportunity to user to fix email

File: pages/rating.vue (hunk 5 above)
Comment (Nainterceptor) [reply pages/rating.vue]: Yes

File: components/form/password.vue [hunk 6]
```diff
@@ -0,0 +1,63 @@
+<template>
+  <div>
+    <input
+      v-validate="'required'"
+      v-bind="$attrs"
```
Comment (Rodrive) [inline components/form/password.vue]: Same thing here : You have not disabled attrsInheritance so every attrs will be on `<div> `and `<input>` except style and class. For exemple you are calling this component with an id, this id will be on div and input

Comment (Nainterceptor) [reply components/form/password.vue]: Done !

**Pull request merged**